client/src/app/hooks/useCodeUrlCheck.ts

```diff
@@ -11,15 +11,24 @@ export function useCodeUrlCheck() {
   const codeInput = useCodeInput();
 
   useEffect(() => {
-    const code = location.pathname.slice(1);
-    if (!validateCode(code)) {
-      codeInput?.setValue(code);
-      codeInput?.setSubmitting(true);
+    if (location.pathname == "/" && location.hash) {
       navigate("/r", { replace: true });
-      // HACK: have a better way to wait for wormhole to initialize
-      setTimeout(() => {
-        wormhole?.saveFile(code);
-      }, 2000);
+
+      const code = location.hash.slice(1);
```

Comment:
I think we need slide 2 symbols as I copy paste url http://localhost:8080/#/3-atlantic-crucial, I get
<img width="474" alt="image" src="https://user-images.githubusercontent.com/104410044/189673609-f3949574-78ab-4490-a579-b54ce7437d4b.png">

Reply:
The last `/` has been dropped. The url you receive should be http://localhost:8080/#3-atlantic-crucial now.